.gitignore

```diff
@@ -1,2 +1,3 @@
+# .gitignore
 node_modules/
```

Comment:
- [ ] Great job adding test.md to your `.gitignore` file. However you haven't a created a `test.md` file as suggested by the previous reviewer. Please do so.